.github/workflows/superset-e2e.yml

```diff
@@ -20,12 +34,12 @@ jobs:
       pull-requests: read
     strategy:
       # when one test fails, DO NOT cancel the other
-      # containers, because this will kill Cypress processes
+      # parallel_id, because this will kill Cypress processes
       # leaving the Dashboard hanging ...
       # https://github.com/cypress-io/github-action/issues/48
       fail-fast: false
       matrix:
-        containers: [1, 2, 3]
+        parallel_id: [0, 1, 2, 3, 4, 5]
```

Comment:
FWIW, since the setup for these runners takes a while, I was keeping the number of workers low enough to see a higher ROI on parallelization. This might shave off a minute or two, but will burn twice the coal.

Reply:
right, there's like 7 minutes of setup and in aggregate maybe 45 minutes of parallelizable stuff. At 6 workers it's about 50/50 of the time on overhead VS work. I thought bumping this up made some sense as it has now become the bottleneck as the longest CI workflow.

Reply:
One thing that could be nice would be to have a single worker prepare some stuff for the matrix as a whole, so that overhead could remain 7 minutes, but not be done on N workers... Not sure what GHA offers around that.